ctags/parsers/python.c

```diff
@@ -825,5 +859,6 @@ extern parserDefinition *PythonParser (void)
 	def->kindCount = ARRAY_SIZE (PythonKinds);
 	def->extensions = extensions;
 	def->parser = findPythonTags;
+	def->useCork = true;
 	return def;
 }
```

Comment:
Yes, exactly, the new parser should be used. I just didn't want want to add one more big diff into this patch.

Will fix the above minor issues though.